cli/run_util.go

```diff
@@ -324,13 +324,7 @@ func (r WorkflowRunner) activateStep(
 		specStep, err := bitrise.ReadSpecStep(stepYMLPth)
 		log.Debugf("Spec read from YML: %#v", specStep)
 		if err != nil {
-			ymlPth := stepYMLPth
-			if origStepYMLPth != "" {
-				// in case of local step (path:./) we use the original step definition path,
-				// instead of the activated step's one.
-				ymlPth = origStepYMLPth
-			}
-			err = fmt.Errorf("failed to parse step definition (%s): %s", ymlPth, err)
+			err = fmt.Errorf("parse step.yml of '%s': %s", stepIDData.IDorURI, err)
```

Comment:
I think in case of `path::` step references it's enough to print the reference, it's a relative path to a file I have access anyway. I don't think it's worth creating a leaky abstraction (in the `ActivatedStep` struct) just to print this local file path.